### ISSUE TITLE
ghq fails to clone or list if files or directories in the GHQ_ROOT are not accessable

### ISSUE THREAD

**Issue by uberjay** [motemen/ghq#173]
This looks similar to #165, but instead of a dangling symlink, I have directories which are root owned. It's probably a little odd, but I have my GHQ_ROOT set to the root of a macOS volume. In there are several root owned directories (`.Trashes`, `.Spotlight-V100` and `.fseventsd`).

In my opinion, skipping over inaccessible directories, or _possibly_ directories starting with a dot would be a good idea. As is, after updating my ghq, no operations work anymore:

```
ghq get https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git
     error open /Volumes/code/.Spotlight-V100: permission denied
```

After disabling spotlight indexing and removing that directory, the next one is .Trashes:

```
$ ghq list
     error failed to filter repos while walkLocalRepositories(repo): open /Volumes/code/.Trashes: permission denied
```

**Comment by Songmu** [comment]
Thanks for reporting. I fixed at #176.

Instead of ignoring all dotfiles flatly, I made a check if it is a dotfile if there is no permission.

**Comment by Songmu** [comment]
released as v0.12.3. please try it.
